Jenkinsfile

```
@@ -660,9 +660,11 @@ def withBeatsEnv(boolean archive, Closure body) {
   ]) {
     deleteDir()
     unstash 'source'
-    dockerLogin(secret: "${DOCKERELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
-    // FIXME workaround untill we fix the packer cache
-    sh 'docker pull docker.elastic.co/observability-ci/database-enterprise:12.2.0.1'
+    if(os == 'linux'){
+      dockerLogin(secret: "${DOCKERELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
+      // FIXME workaround untill we fix the packer cache
+      sh 'docker pull docker.elastic.co/observability-ci/database-enterprise:12.2.0.1'
```

NIT: avoid DDOS  or so
```suggestion
      // Retry to avoid DDoS detection from the server
      retry(3) {
        sleep randomNumber(min: 2, max: 5)
        sh 'docker pull docker.elastic.co/observability-ci/database-enterprise:12.2.0.1'
      }
```